src/components/RecipeEdit.vue

```diff
@@ -201,20 +271,25 @@ export default {
     mounted () {
         // Store the initial recipe configuration for possible later use
         if (this.recipeInit === null) {
-            this.recipeInit = this.recipe
+            this.recipeInit = Object.assign({}, this.recipe)
```

Comment:
@sam-19 I think this should circumvent the "passing by reference" you mentioned [here](https://github.com/nextcloud/cookbook/pull/378#issuecomment-724768627)

Reply:
Yes, I didn't notice there were two separate PRs. This will probably be enough.

Reply:
In this PR I actually require the initial recipe state to check if the current recipe state has changes. The other PR is just a small fix to enable recipe creation at all until this PR is reviewed and accepted.